RoddyCore/src/de/dkfz/roddy/ExitReasons.groovy

```diff
@@ -25,9 +25,11 @@ enum ExitReasons {
     analysisNotLoadable(244, "Could not load analysis"),
     severeConfigurationErrors(243, "Severe configuration errors occurred."),
     unhandledException(242, "Unhandled exception."),
-    unknownSSHHost(241, "Unknown SSH host."),
+    unknownSSHHost(241, "Could not setup SSH access with your configuration: The specified host is not available ."),
     invalidSSHConfig(240, "SSH setup is not valid."),
     fatalSSHError(239, "Fatal error during SSH setup."),
+    aJobHadAnError(238, "At least one of your started jobs exited with an error."),
```

Comment:
"Job exited with error". Obviously the job was started and it obviously was the job of the person that submitted it and is it really that important here that we do not know whether it is more than one job?

Reply:
This message does not get printed. It is more documentary. The user will see a list of finished jobs with their state (successful, not successful). I'll change it though.

Reply:
We are also users. I admit I do not like to look into the logs, because they are too verbose and it is hard to find things.